TestingTravis/TestClass.swift

```diff
@@ -17,6 +17,7 @@ class TestClass: NSObject {
         print(forceUnwrapMe!)
         print(forceUnwrap2!)
         //testing code owners
+        //test 2
         let unused = 0
     }
     
```

Comment:
<table data-meta="generated_by_1">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/671f6e74a3b8377764898e0c1206c901b12d976e/TestingTravis/TestClass.swift#L23"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TestClass.swift:23</code></td>
    </tr>
  </tbody>
</table>